pymc_experimental/model_builder.py

```diff
@@ -187,22 +192,29 @@ def load(cls, fname):
 
         filepath = Path(str(fname))
         idata = az.from_netcdf(filepath)
-        self = cls(
-            json.loads(idata.attrs["model_config"]),
-            json.loads(idata.attrs["sampler_config"]),
-            idata.fit_data.to_dataframe(),
+        if "sampler_config" in idata.attrs:
+            sampler_config = json.loads(idata.attrs["sampler_config"])
+        else:
+            sampler_config = {}
+        model_builder = cls(
+            model_config=json.loads(idata.attrs["model_config"]),
+            sampler_config=sampler_config,
```

Comment:
```suggestion
            sampler_config=json.loads(idata.attrs["sampler_config"]),
```

Reply:
same as the previous one, I wrongly assumed that in case the user wouldn't pass the config we'd just fill it with empty one, ignoring the fact that actually default value of config should be preserved in the saved form of model